pkg/cmd/serviceaccount/svcaccountcmdutil/credentials/credentials.go

```diff
@@ -13,10 +13,11 @@ import (
 )
 
 const (
-	EnvFormat        = "env"
-	JSONFormat       = "json"
-	PropertiesFormat = "properties"
-	SecretFormat     = "secret"
+	EnvFormat            = "env"
+	JSONFormat           = "json"
+	PropertiesFormat     = "properties"
+	SecretFormat         = "secret"
+	JavaPropertiesFormat = "java-kafka-properties"
```

Comment:
I would just call it `java-properties`

Reply:
That did occur to me. But since `.properties` file are primarily used for java-related technologies, this name sounded redundant.

Reply:
This is java-kafka specific, so the name you've chosen seems reasonable.

Reply:
Thanks for the verifying.